props/prop_base.h

```diff
@@ -457,6 +502,22 @@ class PropBase : CommandParser, Looper, protected SaberBase {
     SetPreset(current_preset_.preset_num - 1, true);
   }
 
+  // Go to the previous Preset skipping NewFont and Preon effects using FastOn.
+  void previous_preset_fast() {
+#ifdef SAVE_PRESET
+    SaveState(current_preset_.preset_num - 1);
+#endif
+    UpdatePreset(current_preset_.preset_num - 1);
+  }
+
+  // Go to first Preset.
+  void first_preset() {
```

Comment:
Why would anybody need this?
Also, if this is a "fast" function, it ought to be called "first_preset_fast".

Reply:
Keep it in your prop for now.